backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceTypeDisease.java

```diff
@@ -27,6 +28,7 @@ public class DeviceTypeDisease extends IdentifiedEntity {
 
   @Column private String testPerformedLoincCode;
   @Column private String equipmentUid;
+  @Column @Setter private String equipmentUidType;
```

Comment:
Couldn't figure out another way to access the update for the `DeviceTypeDisease` models so added a setter field here. Happy to go back and remove it after this one-time update since theoretically the field will be included on device creation from here out.